cassandra/start_cassandra.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env bash
```

Comment:
I wonder whether this should actually live in clearwater-cassandra (and be caled start_cassandra_foregrounded), but the /usr/bin location is fine.